src/main/java/training/commands/ArmCommands/rollerBackward.java

```diff
@@ -0,0 +1,21 @@
+package training.commands.ArmCommands;
+
+import training.subsystems.ArmSubsystems.Roller;
+
+public class rollerBackward {
```

Comment:
```suggestion
public class RollerBackward {
```
also the name not sound like an action, maybe RollBackward